test/conformance/exp_command_buffer/buffer_fill_kernel_update.cpp

```diff
@@ -14,10 +14,30 @@ struct BufferFillCommandTest
         UUR_RETURN_ON_FATAL_FAILURE(
             urUpdatableCommandBufferExpExecutionTest::SetUp());
 
-        // First argument is buffer to fill (will also be hidden accessor arg)
-        AddBuffer1DArg(sizeof(val) * global_size, &buffer);
+        ASSERT_SUCCESS(urMemBufferCreate(context, UR_MEM_FLAG_READ_WRITE,
+                                         sizeof(val) * global_size, nullptr,
+                                         &buffer));
+
+        // TODO - Enable single code path after https://github.com/oneapi-src/unified-runtime/pull/1176
```

Comment:
Maybe a good idea to create an issue for this?

Reply:
We don't tend to look at the UR GitHub issues, but removed this TODO and will track it in our team internally instead.